apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1682,6 +1682,14 @@ class LivechatClass {
 		return false;
 	}
 
+	async afterAgentUserActivated(user: IUser) {
+		if (!user.roles.includes('livechat-agent')) {
+			throw new Meteor.Error('invalid-user-role');
```

Comment:
```suggestion
			throw new Error('invalid-user-role');
```
Couldn't we just throw a normal error here to avoid using more Meteor?